src/config/mod.rs

```diff
@@ -286,16 +273,22 @@ impl AgentConfig {
         None
     }
 
-    pub fn get_documentation() -> Result<String, AgentConfigError> {
+    fn default_hostname() -> anyhow::Result<String> {
+        hostname::get()?
+            .into_string()
+            .map_err(|_| anyhow::anyhow!("invalid utf-8 hostname string"))
+    }
+
+    pub fn get_documentation() -> String {
         let mut doc_string = String::new();
         for option in AgentConfig::get_options() {
-            doc_string.push_str(&format!("\n\n\n### {}\n\n", option.name));
+            doc_string.push_str(&format!("\n\n\n=== {}\n\n", option.name));
             doc_string.push_str(&format!(
                 "*Default value*: {}\n\n",
```

Comment:
Not sure but maybe this makes sense:

```suggestion
                "*Default value*: `{}`\n\n",
```